Makefile

```diff
@@ -28,6 +28,15 @@ ifndef AWS_IAM_ARN
 	$(error AWS_IAM_ARN is undefined)
 endif
 
+.PHONY: .check-ou-mapping-configmap-env
+check-ou-mapping-configmap-env:
+ifndef OSD_STAGING_1_OU_ROOT_ID
```

Comment:
What should these be set to?

Reply:
The ID of the root and base (location where new OUs are created) OUs.
2 page in the [design doc](https://docs.google.com/document/d/1IK3M32J5nQkS7eyZ-npU2dSQJxMdyfrFAPra4_SBScM/edit#)